src/Chapter3.hs

```diff
@@ -343,6 +343,12 @@ Define the Book product data type. You can take inspiration from our description
 of a book, but you are not limited only by the book properties we described.
 Create your own book type of your dreams!
 -}
+data Book = Book
+  { title           :: String,
+    author          :: String,
+    tableOfContents :: [Int]
```

Comment:
Interesting idea 💡 

We usually suggest to name fields with the prefixes, like `bookTitle`, so there won't be any problems with the duplicate functions in the module 🙂 But this is just FYI!